dvm.sh

```diff
@@ -60,6 +72,8 @@ if [ -z "$DVM_DIR" ]; then
 fi
 unset DVM_SCRIPT_SOURCE 2> /dev/null
 
+DVM_VERSION_DIR="${DVM_DIR}/bin/docker/"
```

Comment:
We could probably get away without it, but it lets us not repeat the `/bin/docker/` bit, or change it later in one place.